test/sass/_sections.scss

```diff
@@ -13,15 +13,15 @@
 
       @include expect {
         // sass-lint:disable-all
-        border-bottom: 0.44444rem solid #e07;
+        border-bottom: size('border') solid #e07;
```

Comment:
All the config is available. Sometimes I would use it, sometimes I wouldn't. Basically:

- If I want to be sure that the output is an exact value, I use the value - `0.44444rem`
- If I want to be sure that the output matches a pattern, I'll use the pattern - `size('border')`

In this case, we just want to know that it's using that pattern. This way we don't have to change the test every time the pattern change.